### PR TITLE
Built reviews components: Controller, Model, and new View

### PR DESCRIPTION
**Pull request by mgraonic**
Finished Reviews. Still need tests for review logic.

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class ReviewsController < ApplicationController
+belongs_to :product
+
+validates :rating, presence: true, numericality: true, length: { in: 1..5 }
+validtes :description, presence: true
```
Comment (Krashaune) [inline app/controllers/reviews_controller.rb]: grammar validates

File: test/models/review_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+describe Review do
```
Comment (Krashaune) [inline test/models/review_test.rb]: Missing model tests

Comment (brandyaustinseattle) [reply test/models/review_test.rb]: Hey the model tests are under test/models/order_test.rb

Where's the review_test coming from?

Comment (Krashaune) [reply test/models/review_test.rb]: Disregard my comment I just realized the pull request message states that there's no tests.

**Pull request merged**